draft-ietf-rtcweb-jsep.xml

```diff
@@ -741,66 +741,82 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
+          including cases in which the track changes its resolution,
+          or is replaced with a different track.
+          </t>
+
+          <t>Depending on how the RtpSender is configured, it may be producing
+          a single encoding at a certain resolution, or, if
+          simulcast <xref target="sec.simulcast" /> has been negotiated,
+          multiple encodings, each at their own specific resolution. In
+          addition, depending on the configuration, each encoding may have the
+          flexibility to reduce resolution when needed, or may be locked to
+          a specific output resolution.
+          </t>
+
+          <t>For each encoding being produced by the RtpSender, the following
+          rules are applied to determine what should be transmitted:
+            <list style="symbols">
+              <t>First, the right "a=imageattr recv" attribute is selected.
+              This is performed by taking the
+              attribute with the highest "q=" value from the set of attributes
+              that reference the media format that has been selected for the
+              specified encoding. If multiple attributes have the same "q="
+              value, the one that appears first in the m= section is used.
+              Note that while JSEP endpoints will include at most
+              one "a=imageattr recv" attribute per media format, JSEP
+              endpoints may receive session descriptions from non-JSEP
+              endpoints with m= sections that contain multiple such attributes.
+              </t>
+
+              <t>If there is an applicable "a=imageattr recv" attribute for
+              the encoding, the limits from the attribute are then compared to
+              the encoder resolution. Only the specific limits mentioned below
+              are considered; any other values, such as picture aspect ratio,
+              MUST be ignored.
+              Note that when considering a MediaStreamTrack that is producing
+              rotated video, the unrotated resolution MUST be used for the
+              checks. This is
+              required regardless of whether the receiver supports
+              performing receive-side rotation (e.g., through CVO
+              <xref target="TS26.114" />), as it significantly simplifies
+              the matching logic.</t>
+
+              <t>If the attribute includes a "sar=" (sample aspect ratio)
+              value set to something other than "1.0", indicating the
+              receiver wants to receive non-square pixels, this cannot be
+              satisfied and the sender MUST NOT transmit the encoding.</t>
+
+              <t>If the encoder resolution exceeds the maximum size permitted by the
+              attribute, and the encoder is allowed to adjust its resolution,
+              the encoder SHOULD apply downscaling in order to satisfy the limits,
+              although the downscaling MUST NOT change the track aspect ratio.
```

Comment:
Do you mean the picture aspect ration for this particular track? "track aspect ratio" is not clear on what is referred to.